### PR TITLE
Fix Dummy discrimination

### PR DESCRIPTION
**Pull request by Jacfomg**
Fix the way the dummy generates random numbers for Discrimination.

When Singleshot it gets either gets 0 or 1 and when Cylic a float in [0,1].

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

File: src/qibolab/instruments/dummy.py [hunk 1]
```diff
@@ -73,7 +76,10 @@ def sweep(
 
         for ro_pulse in sequence.ro_pulses:
             if options.acquisition_type is AcquisitionType.DISCRIMINATION:
-                values = np.random.rand(expts)
+                if options.averaging_mode is AveragingMode.SINGLESHOT:
+                    values = np.random.randint(2, size=expts)
+                elif options.averaging_mode is AveragingMode.CYCLIC:
+                    values = np.random.rand(expts)
```
Comment (AleCandido) [inline src/qibolab/instruments/dummy.py:82]: It's twice the same, so it is worth to make a tiny function for this :)

**Review by rodolfocarobene — APPROVED** [review]
Thanks @Jacfomg, to me looks ok, if you want you could also change `expts` to something more understable this could be the right moment

**Comment by AleCandido** [comment]
P.S.: with a dedicated function, you could even test it in isolation, making Codecov happier :+1:

**Comment by Jacfomg** [comment]
> Thanks @Jacfomg, to me looks ok, if you want you could also change `expts` to something more understable this could be the right moment

As its creator what do you suggest ? `exp_points` ?

File: src/qibolab/instruments/dummy.py (hunk 1 above)
Comment (Jacfomg) [reply src/qibolab/instruments/dummy.py:82]: I think the whole loops for play and sweep are, I will create a function for both

**Comment by rodolfocarobene** [comment]
> As its creator what do you suggest ? exp_points ?

As you saw, I am awful with names :-). `exp_points` is good for me, some other options could be `num_points`, `points`, `result_len`...
Everything is good to me

**Comment by Jacfomg** [comment]
> > As its creator what do you suggest ? exp_points ?
> 
> As you saw, I am awful with names :-). `exp_points` is good for me, some other options could be `num_points`, `points`, `result_len`... Everything is good to me

Same here hahaha

**Review by AleCandido — APPROVED** [review]
Even better than what I asked for!

Thanks @Jacfomg

**Pull request merged**